spec/controllers/wishlists_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
```

Comment:
Nothing important, but You can change type of qutation marks to be the same in the whole file.